intro-to-linux/PITCHME.md

```diff
@@ -4,19 +4,48 @@
 
 --- 
 ## What is Linux ?
-Linux is an open source operating system (OS).
-Every Linux-based OS involves the Linux kernel
+Linux is an open source operating system (OS). 
+
+A good start to the class would be asking the students if they know what Linux is, although they may be quiet at first, we found that the more engaging a class is, the more keen they are to learn too.
+
+* What are other famous open source operating systems?
+* What is an operating system?
+* Do they use an OS?
+
+
+Every Linux-based OS involves the Linux kernel.
+
+THat's the main component of the Linux operating system, communicates between a computer's hardware and its processes.
+
+A nice analogy to explain the role of the kernel:
+
+
+```
+The kernel is a busy personal assistant for a powerful executive (the hardware).
+
+It’s the assistant’s job to relay messages and requests (processes) from employees and the public (users) to the executive, to remember what is stored where (memory), and to determine who has access to the executive at any given time and for how long.
+```
 
 
-+++
 ## Linux and Open source
-Anyone can run, study and modify and redistribute the linux source code
+Anyone can run, study and modify and redistribute the Linux source code.
+
 
-+++
 ## Linux Distributions
+
+Linux distribution = complete Linux system package
+
+Different distros can be customized for specific user gorups, for example software enginners or audio/video editors
+
 ![Linux Distros](/assets/img/linux_kernel.png)  <!-- Copyright freecodecamp -->
 
-+++
+
 ## Command Line
 The command line is your direct access to a computer.
-The command line and open source software, together, give users unrestricted access to their computer
\ No newline at end of file
+
+It's like a text interface to your computer.
+
+The command line and open source software, together, give users unrestricted access to their computer.
+
+* Do any students use the command line?
+* How else do we refer to it? Hint: terminal, shell, console
```

Comment:
Why would we use a text interface?

At first, it might seem that it's old-fashioned and that there must be better ways to work with the computer in 2024.

However, although learning the shell does require some effort, it enables a proficient user to reach a level of control and power over the computer that no graphical tool can match.

Not only are there a great many useful tools immediately available in the terminal, it also provides a way to combine the existing tools into "pipelines" that can accomplish almost anything a power user could want to do.

Learning to build pipelines also teaches fundamental concepts of computer science, as the tools that we combine are by themselves each quite small - the power comes from the combination. One of the core ideas of the Linux / Unix philosophy is: "When making tools for others, have the program do one thing, but do it well". For the user, they can then choose which of the small tools they want to combine, to build up the tool they actually need.